### PR TITLE
Feature/project start

### PR DESCRIPTION
**Pull request by RobertoMSousa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,196 +1,3 @@
-# Logs
```
Comment (Webern) [inline .gitignore]: Shouldn't we keep all of this?

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        "array-bracket-spacing":  true, // space on the square brackets
+        "space-within-parens": [true, 1], // force one sapce on parens
+        "class-name": true,
+        "comment-format": [
+            true,
+            "check-space"
+        ],
+        "indent": [
+            true,
+            "tabs"
```
Comment (Webern) [inline tslint.json]: spaces

File: src/app.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+// compress all the responses
+app.use( compression() );
+
+app.use( bodyParser.json() );
+app.use( bodyParser.urlencoded( { extended: true } ) );
+app.use( expressValidator() );
+
+/**
+ * Primary app routes.
```
Comment (Webern) [inline src/app.ts]: prefer line comments
// Primary app routes.

File: src/app.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import bodyParser from "body-parser";
+import dotenv from "dotenv";
+import path from "path";
+import expressValidator from "express-validator";
+import bluebird from "bluebird";
+
+
+// Load environment variables from .env file, where API keys and passwords are configured
+dotenv.config( { path: ".env.example" } );
+
```
Comment (Webern) [inline src/app.ts]: Avoid extra empty lines.

**Review by webern — CHANGES_REQUESTED** [review]
a few small style requests, thanks!

**Pull request merged**